using-timescaledb/continuous-aggregates.md

```diff
@@ -16,148 +17,202 @@ every refresh. Only new and/or invalidated data will be calculated.  Since this
+unaggregated data in the underlying table (for example, `conditions`).
+
+If you, however, want to just get partially aggregated data and not
+include data from the underlying table, you can set the option
+`timescaledb.materialized_only` to `true` using `ALTER VIEW`:
+
+```sql
+ALTER VIEW conditions_summary_hourly SET (
+    timescaledb.materialized_only = true
+);
+```
+
+>:TIP: You can find the names of all the materialized hypertable by
+>querying `timescaledb_information.continuous_aggregates`.
+>
+> ```sql
+> postgres=# SELECT view_name, materialization_hypertable
+> postgres-#   FROM timescaledb_information.continuous_aggregates;
+>          view_name         |            materialization_hypertable             
+> ---------------------------+---------------------------------------------------
+>  conditions_summary_hourly | _timescaledb_internal._materialized_hypertable_30
+>  conditions_summary_daily  | _timescaledb_internal._materialized_hypertable_31
+> (2 rows)
+> ```
+
+Each continuous aggregate uses the *bucket width* of the `time_bucket`
+(given by the parameter `bucket_width`) function in the view
+definition to decide what to update. Time-series data is typically
+ordered, so it is usually the last bucket that gets most of the
+updates. Recomputing the last bucket when new data arrives negates many
+of the benefits of using continuous aggregation. Older buckets rarely
+get updated and are usually aggregated only once.
+
+As a result, the materialized hypertable lags behind the underlying
+table by one bucket and up to an additional `refresh_interval` as a
+result of the job not running continuously. However, you can change
+this lag to either increase it (or decrease it) by setting the value
+for the `timescaledb.refresh_lag` option of the view and the
+background worker will attempt to materialize everything before
+`refresh_lag` + `bucket_width`.
+
+For example, using `conditions_summary_hourly` above, the bucket width
+is 1 hour, so if we set the `refresh_lag` to 1 hour, the
+materialization will be 2 hours behind the current time.
+
+```sql
+ALTER VIEW conditions_summary_hourly SET (
+  timescaledb.refresh_lag = '1 hour'
+);
+```
+
+Lowering the `refresh_lag` parameter means that the materialization
+table follow inserts more closely, but can cause write amplification
+that might degrade insert performance. You can also set `refresh_lag`
+to negative values, so if you set `refresh_lag` to `-bucket_width`
+it will make the continuous aggregate read all data into the
+materialization table as it is added, but will cause more writes
+into the materailization table, leading to write amplification.
+
+Regardless of the value of `refresh_lag`, the continuous aggregate
+view will not be different. However, changing `refresh_lag` can be an
+advantage if queries on the continuous aggregate are *very* frequent,
+since that would avoid sending repeated queries to the underlying
+table at high frequency.
```

Comment:
I am not sure I understand what this whole paragraph is trying to convey. In particular, I don't understand the point about `sending repeated queries`. Are you referring to the materialization or the queries on the raw data? In any case, I am not sure this is needed and we're too much in the weeds here.

I am guessing that you are trying to say something about the balance between querying raw data and materialized data. I suggest focusing on that aspect to avoid getting bogged down in the details:

`
Please note that the refresh_lag parameter controls the balance between on-the-fly aggregation and pre-computed aggregation when querying the continuous aggregate. In other words, a lower refresh_lag will reduce the amount of on-the-fly aggregation by more aggressive materialization. Keep this in mind when trying to set the right refresh_lag for your application.
`

Reply:
> I am not sure I understand what this whole paragraph is trying to convey. In particular, I don't understand the point about `sending repeated queries`. Are you referring to the materialization or the queries on the raw data? In any case, I am not sure this is needed and we're too much in the weeds here.

I'm referring to the application query frequency. This description is somewhat redundant but how do we motivate `refresh_lag`? If we do not provide an argument, we will get a lot of questions on what purpose it serves, especially since we are ignoring details of the implementation.
> 
> I am guessing that you are trying to say something about the balance between querying raw data and materialized data. I suggest focusing on that aspect to avoid getting bogged down in the details:
> 
> `Please note that the refresh_lag parameter controls the balance between on-the-fly aggregation and pre-computed aggregation when querying the continuous aggregate. In other words, a lower refresh_lag will reduce the amount of on-the-fly aggregation by more aggressive materialization. Keep this in mind when trying to set the right refresh_lag for your application.`

That might work, let me ponder about it.

Reply:
Tried something along those lines. Asking the user to keep something in mind without giving sufficient information will just be frustrating, so I removed that.